SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -368,13 +369,13 @@ bool USpatialGDKEditorSettings::IsManualWorkerConnectionSet(const FString& Launc
 	}
 
 	const TSharedPtr<FJsonObject>* LoadBalancingField;
-	if (!(*LaunchConfigJsonRootObject)->TryGetObjectField("load_balancing", LoadBalancingField))
+	if (!(*LaunchConfigJsonRootObject)->TryGetObjectField("loadBalancing", LoadBalancingField))
```

Comment:
minor: but camelCase is reserved for local variables only right? This is more like a parameter?

Reply:
This is a string, to match a keyword from a configuration file.

Reply:
My point is should the config file itself by PascalCase or camelCase